### ISSUE TITLE
whitelist not working

### ISSUE THREAD

**Issue by slyfox1186** [dhowe/AdNauseam#2239]
### Describe the issue

This [YouTube](https://youtu.be/TrCrzl0Cq14) video should explain a lot.

### Your settings
- OS/version: Win11 latest to date x64
- Browser/version: Firefox Beta latest to date
- AdNauseam version: latest to date
- Other extensions you have installed: disabled all other extensions and it still was not working
- AdNauseam settings: [Screenshot](https://i.imgur.com/jhVmYi3.jpeg)

##### Your filter lists
All of them + Russian language

### Your location/country
(Atlanta, GA USA)

** For adult or potentially offensive sites/ads, please add a [Content-Warning] tag in the title to warn developers the ticket may contain such material.

**Comment by mneunomne** [comment]
hey @slyfox1186 thanks for submiting the ticket. Maybe the issue is that we didn't message this correctly. 

The link that you have clicked is actually the exception list that adnauseam works with. The whitelist bellow is an additional list of websites that you can add yourself. for example if you want to add "nytimes.com" in the whitelist, you simply add as:

```
nytimes.com
washingtonpost.com
! etc...
```

the list that you tried to add in the video was the exception list that is already enabled by default. If you try to access any of the websites in that list, you'll see that adnauseam icon turns green, meaning it is allowing the ads on that webpage depending on the configurations on your initial page:

<img width="425" alt="Screenshot 2022-12-21 at 02 37 16" src="https://user-images.githubusercontent.com/4967860/208800084-b03c81b1-f675-47c2-bd42-e3240a316f83.png">

So if you want to disable this initial EFF whitelist, you should uncheck the two checkboxes above.

If you want to add aditional websites, simply add it to the whitelist following the uBlock Origin guide:

https://github.com/gorhill/uBlock/wiki/How-to-mark-a-web-site-as-trusted

Hope this answers your doubt. Otherwise let us know what you need and we I can give a more precise solution. Either way this probably means we need to change how we phrase our whitelist page.

**Comment by dhowe** [comment]
closing for now - pls create a separate ticket for whitelist language if needed

**Comment by mneunomne** [comment]
I guess the issue is rather that the DNT EFF info maybe shouldn't be displayed in the whitelist section, but rather where the DNT options are. But not sure how to fit it there.

_[EFF DNT Policy Whitelist‎](chrome-extension://dkoaabhijcomjinndlgbmfnmnjnmdeeb/asset-viewer.html?url=eff-dnt-whitelist) [(www.eff.org)](https://www.eff.org/dnt-policy)_

**Comment by dhowe** [comment]
@mneunomne can you explain? the DNT list is a whitelist, and it has its own section on the whitelist page to differentiate from user-supplied sites

**Comment by mneunomne** [comment]
All I am saying is that perhaps it is a bit confusing that the DNT whitelist is actually a filter list, and has a different syntax from whitelist.

![Screen Shot 2022-12-28 at 12 43 52](https://user-images.githubusercontent.com/4967860/209784627-f3292293-0365-4fe9-994c-1478c685dbbf.png)

Maybe it would make more sense for it to show up on the filter list on a DNT section? But maybe there is a reason why it isn't there.

**Comment by dhowe** [comment]
Yes, we can consider whether it should have its own page, or, as [suggested by Mushon](https://github.com/dhowe/AdNauseam/issues/2124), we replace it with GPC or similar. 

But can you verify that unclicking that list in the whitelist settings disables the two DNT settings ?
